.readthedocs.yml

```diff
@@ -8,6 +8,7 @@ build:
   jobs:
     post_create_environment:
       - "conda env export"
+      - PLOOMBER_VERSION_CHECK_DISABLED=true
```

Comment:
this won't work since it'll set the variable after the documentation is build. I think we want the `pre_build` job: https://docs.readthedocs.io/en/stable/build-customization.html

let's see if this works


pre_build:

```
export PLOOMBER_VERSION_CHECK_DISABLED=true
```


then add one test notebook to the documentation with this code in the first cell:

```python
from os import environ
var = environ.get("PLOOMBER_VERSION_CHECK_DISABLED")
print(var)
```

this will let us see if the environment variable is correctly set